index.html

```diff
@@ -35,9 +35,157 @@ <h1 class="headline_top">I'm Sergiy<br>Glad to see you!</h1>
+        </div>
+        <div class="work-item"></div>
+        <div class="work-item"></div>
+        <div class="work-item"></div>
+      </div>
+    </section>
+    <!-- work 3 -->
+    <section> 
+      <div class="work-container">
+        <div class="work-item">
+          <section class="work-item-flex">
+            <img src="./img/work-snapshot3.png" alt="work3">
+            <p class="work-item-title">Tonic</p>
+            <ul class="work-item-frame">
+              <li class="work-item-frame-client">CANOPY</li>
+              <li class="work-item-frame-counter">●</li>
+              <li class="work-item-frame-role">Back End Dev</li>
+              <li class="work-item-frame-counter">●</li>
+              <li class="work-item-frame-year">2015</li>
+            </ul>
+            <p class="work-item-text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="work-item-tags">
+              <li class="work-item-tag">html</li>
+              <li class="work-item-tag">css</li>
+              <li class="work-item-tag">javaScript</li>
+            </ul>
+            <button class="work-item-button">See Project</button>
+          </section>
+        </div>
+        <div class="work-item"></div>
+        <div class="work-item"></div>
+        <div class="work-item"></div>
+      </div>
+    </section>
+    <!-- work 4 -->
+    <section>
+      <div class="work-container">
+        <div class="work-item">
+          <section class="work-item-flex">
+            <img src="./img/work-snapshot4.png" alt="work4">
+            <p class="work-item-title">Multi-Post Stories</p>
+            <ul class="work-item-frame">
+              <li class="work-item-frame-client">CANOPY</li>
+              <li class="work-item-frame-counter">●</li>
+              <li class="work-item-frame-role">Back End Dev</li>
+              <li class="work-item-frame-counter">●</li>
+              <li class="work-item-frame-year">2015</li>
+            </ul>
+            <p class="work-item-text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="work-item-tags">
+              <li class="work-item-tag">html</li>
+              <li class="work-item-tag">css</li>
+              <li class="work-item-tag">javaScript</li>
+            </ul>
+            <button class="work-item-button">See Project</button>
+          </section>
+        </div>
+        <div class="work-item"></div>
+        <div class="work-item"></div>
+        <div class="work-item"></div>
+      </div>
+    </section>
+    <!-- About -->
+    <section class="about">
+      <h1 class="headline-top">About<br>Myself</h1>
+      <p class="headline-text">Hello I'm a software developer! I can helpyou build a product , feature or website
+        <br>Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.
+      </p>
```

Comment:
Thank you! Done, margins were added.

Reply:
Kindly, I suggest you review these design details so that your site more closely resembles the original Figma design.
![image](https://user-images.githubusercontent.com/41912912/191968602-d7a72664-094a-4c3b-a597-f709cc5dbb47.png)

Reply:
@mgmediaweb Yes, I didnt see this. Now I've implemented it. Thanks!